stetho/src/main/java/com/facebook/stetho/inspector/network/NetworkEventReporterImpl.java

```diff
@@ -152,6 +152,22 @@ public void responseHeadersReceived(InspectorResponse response) {
           getResourceTypeHelper().determineResourceType(contentType) :
           Page.ResourceType.OTHER;
       receivedParams.response = responseJSON;
+
+      AsyncPrettyPrinterRegistry registry= peerManager.getAsyncPrettyPrinterRegistry();
+      AsyncPrettyPrinterFactory factory = null;
```

Comment:
if we're not using this outside of the loop, then we can just declare it inside the loop, e.g.
`AsyncPrettyPrinterFactory factory = registry.lookup(response.headerName(i));`